python/ctsm/modify_fsurdat/fsurdat_modifier.py

```diff
@@ -65,10 +65,15 @@ def fsurdat_modifier(cfg_path):
     landmask_file = get_config_value(config=config, section=section,
         item='landmask_file', file_path=cfg_path, can_be_unset=True)
 
+    # Create ModifyFsurdat object
+    modify_fsurdat = ModifyFsurdat.init_from_file(fsurdat_in,
+        lnd_lon_1, lnd_lon_2, lnd_lat_1, lnd_lat_2, landmask_file)
```

Comment:
Moved this up from a few lines down so as to eliminate some "magic" numbers that I could instead read from the fsurdat file.